### PR TITLE
Contributing to docs: style tweaks

### PR DESCRIPTION
**Pull request by c0ffeeca7**

### CODE REVIEW

File: source/docs/contribute/documentation.md [hunk 1]
```diff
@@ -2,52 +2,52 @@
 title: Documentation
 description: "Documentation development"
 ---
-The documentation site for HACS is build with [mkdocs-material](https://squidfunk.github.io/mkdocs-material/) and hosted on [Cloudflare Pages](https://pages.cloudflare.com/)
+The documentation site for HACS is using [markdown](https://daringfireball.net/projects/markdown/) and built with [mkdocs-material](https://squidfunk.github.io/mkdocs-material/). The site is hosted on [Cloudflare Pages](https://pages.cloudflare.com/).
```
[the remark this answers is not in the document]
Comment (ludeeus) [reply source/docs/contribute/documentation.md:5]: There is a link to it.

File: source/docs/contribute/documentation.md [hunk 2]
```diff
@@ -56,22 +56,26 @@ source/assets/images/
 
 ## Contents linking
 
-### Reference another page
+Instructions how to create references in markdown.
```
Comment (ludeeus) [inline source/docs/contribute/documentation.md:59]:
```suggestion
Instructions on how to create references in markdown.

```

File: source/docs/contribute/documentation.md (hunk 1 above)
Comment (c0ffeeca7) [inline source/docs/contribute/documentation.md:5]:
```suggestion
The documentation site for HACS is using [Markdown](https://daringfireball.net/projects/markdown/) and is built with [mkdocs-material](https://squidfunk.github.io/mkdocs-material/). The site is hosted on [Cloudflare Pages](https://pages.cloudflare.com/).

```

**Review by ludeeus — APPROVED** [review]
Thanks 👍

**Pull request merged**